historical-adherence-query/java/Main.java

```diff
@@ -1,64 +1,55 @@
 import com.google.common.base.Stopwatch;
 import com.mypurecloud.sdk.v2.ApiClient;
 import com.mypurecloud.sdk.v2.ApiException;
+import com.mypurecloud.sdk.v2.ApiResponse;
 import com.mypurecloud.sdk.v2.Configuration;
 import com.mypurecloud.sdk.v2.PureCloudRegionHosts;
 import com.mypurecloud.sdk.v2.api.UsersApi;
 import com.mypurecloud.sdk.v2.api.WorkforceManagementApi;
-import com.mypurecloud.sdk.v2.extensions.notifications.NotificationHandler;
-import com.mypurecloud.sdk.v2.extensions.notifications.NotificationListener;
-import com.mypurecloud.sdk.v2.model.User;
-import com.mypurecloud.sdk.v2.model.UserMe;
-import com.mypurecloud.sdk.v2.model.WfmHistoricalAdherenceCalculationsCompleteTopicWfmHistoricalAdherenceCalculationsCompleteNotice;
-import com.mypurecloud.sdk.v2.model.WfmHistoricalAdherenceQuery;
-import com.mypurecloud.sdk.v2.model.WfmHistoricalAdherenceResponse;
+import com.mypurecloud.sdk.v2.model.*;
```

Comment:
Being fussy here but maybe it's better to leave it explicitly importing all classes so people know exactly where they're coming from.

Reply:
Hi Ronan, thanks for the suggestion, I reverted it to use the explicit imports.